packages/picasso.js/src/core/chart-components/point/point.js

```diff
@@ -127,15 +127,22 @@ function getType(s) {
   return [type, props];
 }
 
-function createDisplayPoints(dataPoints, { width, height }, pointSize, shapeFn) {
+function createDisplayPoints(dataPoints, { width, height }, pointSize, shapeFn, errorShape) {
   return dataPoints
     .filter((p) => p.show !== false && !isNaN(p.x + p.y))
     .map((p) => {
       let s = p;
       let size = PX_RX.test(p.size) ? parseInt(p.size, 10) : pointSize.min + s.size * (pointSize.max - pointSize.min);
       if (notNumber(size)) {
-        s = DEFAULT_ERROR_SETTINGS.errorShape;
-        size = pointSize.min + s.size * (pointSize.max - pointSize.min);
+        s = {
+          fill: p.fill,
+          stroke: p.stroke,
+          strokeWidth: p.strokeWidth,
+          strokeDasharray: p.strokeDasharray,
+          opacity: p.opacity,
+          ...(errorShape || DEFAULT_ERROR_SETTINGS.errorShape),
```

Comment:
This solution requires the `size` property to exist. Which it should not require.

Reply:
```js
// This wont work
{
 type: 'point',
 settings: { errorShape: { shape: 'saltire' } }
}

// This do work
{
 type: 'point',
 settings: { errorShape: { shape: 'saltire', size: '10px' } }
}
```

Reply:
Ideally from an API point of view, the errorShape properties should be resolved per data point. So that each data point could be configured uniquely, as all other data properties can.